docs/spec/spec.md

```diff
@@ -84,6 +75,7 @@ medical-center
 
 Behaviour of the `init` command,
 - User should invoke the command within a bal project
+- User can use the optional environment variables to indicate the preferred module name and data store, otherwise default values will be used.
```

Comment:
```suggestion
- User can use the optional command options to indicate the preferred module name and data store, otherwise default values will be used.
```